CMakeLists.txt

```diff
@@ -427,7 +427,7 @@ endif()
 set(SCCLASSLIB_EXCLUDE_REGEX "IGNOREME")
 
 if(NOT SC_QT)
-	set(SCCLASSLIB_EXCLUDE_REGEX "${SCCLASSLIB_EXCLUDE_REGEX}|QtCollider")
+	set(SCCLASSLIB_EXCLUDE_REGEX "${SCCLASSLIB_EXCLUDE_REGEX}|QtCollider|GUI")
```

Comment:
Thanks so much! IIUC you could remove `QtCollider` from this regex entirely. There aren't any folders named that in the class lib directory, this might be a remnant from pre-3.7 (https://github.com/supercollider/supercollider/tree/3.6/SCClassLibrary/QtCollider)